### PR TITLE
New metrics process.cpu.percent added

### PR DESCRIPTION
**Pull request by bhogayatakb**
process.cpu.time added with gopsutil lib

### CODE REVIEW

**Review by meenal-developer — APPROVED** [review]

File: receiver/hostmetricsreceiver/internal/scraper/processscraper/internal/metadata/generated_metrics.go [hunk 1]
```diff
@@ -115,6 +119,59 @@ type metricProcessCPUTime struct {
...
+	data     pmetric.Metric // data buffer for generated metric.
+	settings MetricSettings // metric settings provided by user.
+	capacity int            // max observed number of data points added to the metric.
+}
+
+// init fills process.cpu.percent metric with initial data.
+func (m *metricProcessCPUPercent) init() {
+	m.data.SetName("process.cpu.percent")
+	m.data.SetDescription("Total CPU Percent.")
+	m.data.SetUnit("s")
```
Comment (dixudx) [inline receiver/hostmetricsreceiver/internal/scraper/processscraper/internal/metadata/generated_metrics.go]: I did not get the unit here. Does `s` mean seconds or something else ?

Comment (bhogayatakb) [reply receiver/hostmetricsreceiver/internal/scraper/processscraper/internal/metadata/generated_metrics.go]: Yes, percent should have a different unit, thanks for pointing it out

**Comment by bhogayatakb** [comment]
Also, I think CPU Percent should be a Gauge instead of Sum, right ? @dixudx

**Comment by dixudx** [comment]
> Also, I think CPU Percent should be a Gauge instead of Sum, right ? @dixudx

Yes, correct.

File: receiver/hostmetricsreceiver/internal/scraper/processscraper/internal/metadata/generated_metrics.go (hunk 1 above)
Comment (bhogayatakb) [reply receiver/hostmetricsreceiver/internal/scraper/processscraper/internal/metadata/generated_metrics.go]: Changed the unit to "%"

**Comment by bhogayatakb** [comment]
> > Also, I think CPU Percent should be a Gauge instead of Sum, right ? @dixudx
> 
> Yes, correct.

Sum => Gauge conversion done

**Review by dixudx — APPROVED** [review]
Looks good to me.

**Pull request merged**